components/input/input.html

```diff
@@ -0,0 +1,234 @@
+                sandbox="allow-scripts allow-same-origin"
+              >
+              </iframe>
+            </div>
+          </div>
+
+          <hr />
+
+          <div class="secondary-container">
+            <div class="subtitle">Textbox with validation (Warning)</div>
+
+            <div class="subtitle-detail">
+              This style is used to warn the user against a particular
+              situation.
+            </div>
+
+            <div class="comp-demo">
+              <form class="form-group">
+                <div class="input-group input input-warning">
+                  <label class="input-label">Email</label>
+                  <input type="text" placeholder="Type here..." />
+                </div>
+                <div class="input-group input input-warning">
+                  <label class="input-label">Password</label>
+                  <input type="text" placeholder="Type here..." />
+                </div>
+                <div class="input-warning-msg">*Weak password</div>
+              </form>
+            </div>
+
+            <div class="comp-code">
+              <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28196%2C181%2C251%2C1%29&t=shades-of-purple&wt=bw&l=htmlmixed&width=736.25&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=true&fl=1&fm=Fira+Code&fs=14px&lh=143%25&si=false&es=2x&wm=false&code=%253Cform%2520class%253D%2522form-group%2522%253E%250A%2509%253Cdiv%2520class%253D%2522input-group%2520input%2520input-warning%2522%253E%250A%2509%2509%253Clabel%2520class%253D%2522input-label%2522%253EEmail%253C%252Flabel%253E%250A%2509%2509%253Cinput%2520type%253D%2522text%2522%2520placeholder%253D%2522Type%2520here...%2522%2520%252F%253E%250A%2509%253C%252Fdiv%253E%250A%250A%2509%253Cdiv%2520class%253D%2522input-group%2520input%2520input-warning%2522%253E%250A%2509%2509%253Clabel%2520class%253D%2522input-label%2522%253EPassword%253C%252Flabel%253E%250A%2509%2509%253Cinput%2520type%253D%2522text%2522%2520placeholder%253D%2522Type%2520here...%2522%2520%252F%253E%250A%2509%253C%252Fdiv%253E%250A%250A%2520%2520%2509%253Cdiv%2520class%253D%2522input-warning-msg%2522%253E*Weak%2520password%253C%252Fdiv%253E%250A%253C%252Fform%253E"
+                sandbox="allow-scripts allow-same-origin"
+              >
+              </iframe>
+            </div>
+          </div>
+
+          <hr />
+
+          <div class="secondary-container">
+            <div class="subtitle">Textbox with validation (Error)</div>
+
+            <div class="subtitle-detail">
+              This style is used to depict that the data entered is incorrect.
+            </div>
+
+            <div class="comp-demo">
+              <form class="form-group">
+                <div class="input-group input input-error">
+                  <label class="input-label">Password</label>
+                  <input type="text" placeholder="Type here..." />
```

Comment:
Change: Type for password field should be password instead of text. same for confirm password.